Scripts/Audio_Data_Recording.py

```diff
@@ -0,0 +1,127 @@
+# =============================================================================
+# Title: Audio Recording
+# Author: Chidi Ewenike
+# Date: 04/19/2019
+# Organization: Cal Poly CSAI
+# Description: Records audio and labels the recording
+# =============================================================================
+
+
+import wave
+import pyaudio
+import os
+import time
+import winsound
+
```

Comment:
Looks like this file need some style formatting - I can handle it once this is merged in

Reply:
Everything looks good otherwise